cmd/apprepository-controller/controller.go

```diff
@@ -273,35 +273,35 @@ func (c *Controller) syncHandler(key string) error {
 		if errors.IsNotFound(err) {
 			log.Infof("AppRepository '%s' no longer exists so performing cleanup of charts from the DB", key)
 			// Trigger a Job to perfrom the cleanup of the charts in the DB corresponding to deleted AppRepository
-			_, err = c.kubeclientset.BatchV1().Jobs(namespace).Create(context.TODO(), newCleanupJob(name, namespace, c.kubeappsNamespace), metav1.CreateOptions{})
+			_, err = c.kubeclientset.BatchV1().Jobs(namespace).Create(context.TODO(), newCleanupJob(name, namespace, c.conf), metav1.CreateOptions{})
 			return nil
 		}
 		return fmt.Errorf("Error fetching object with key %s from store: %v", key, err)
 	}
 
 	// Get the cronjob with the same name as AppRepository
 	cronjobName := cronJobName(apprepo)
-	cronjob, err := c.cronjobsLister.CronJobs(c.kubeappsNamespace).Get(cronjobName)
+	cronjob, err := c.cronjobsLister.CronJobs(c.conf.Namespace).Get(cronjobName)
 	// If the resource doesn't exist, we'll create it
 	if errors.IsNotFound(err) {
 		log.Infof("Creating CronJob %q for AppRepository %q", cronjobName, apprepo.GetName())
-		cronjob, err = c.kubeclientset.BatchV1beta1().CronJobs(c.kubeappsNamespace).Create(context.TODO(), newCronJob(apprepo, c.kubeappsNamespace), metav1.CreateOptions{})
+		cronjob, err = c.kubeclientset.BatchV1beta1().CronJobs(c.conf.Namespace).Create(context.TODO(), newCronJob(apprepo, c.conf.Namespace, c.conf), metav1.CreateOptions{})
```

Comment:
`newCronJob(apprepo, c.conf.Namespace, c.conf)` --> If you are already passing the conf, why pass the namespace as a different parameter?

Reply:
I completely get your points in having kubeappsNamespace instead of repoNamespace. In the beginning, I saw that both names were being used, so I opted to preserve the most generic one.
I was not aware of https://github.com/kubeapps/kubeapps/pull/2156#discussion_r521683721, so that's my bad, gonna change it asap :)
And, as you mention, having the c.conf makes it possible to avoid passing this parameter again and again.